tests/lib/cmsis_dsp/transform/testcase.yaml

```diff
@@ -47,7 +47,7 @@ tests:
       - CONFIG_CMSIS_DSP_TEST_TRANSFORM_RF32=y
   libraries.cmsis_dsp.transform.cf64:
     min_flash: 1024
-    min_ram: 64
+    min_ram: 128
```

Comment:
> But while executing step test_arm_cfft_f64_cmplx_noisy_4096 there is a malloc() of length:
> 8192 * sizeof(float64_t) = 8192 * 8 = 64k
> but we need more RAM for variables, stack, ...

As you noted in #27849, `min_ram: 64` is indeed insufficient.

I am not sure if setting this to `128` is the best solution though; `80` or `96` should (at least, theoretically) work.

Reply:
I dig a bit a found a nucleo_l476rg (96k ram 1024 flash) able to pass the test.
And I found no board (any manufacturers) with 80k ram and enough flash (1024)
So I propose to set min_ram: 96

Reply:
Split done
set 96 done